ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1271 @@
+{
```

Comment:
<p>In general I don't like that you switch from dataset to models hyperparameters then again to dataset and to model again.</p><p>Consider following structure: data loading - data visualisation - constructor - hyperparameters -...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think that all params should be consolidate in one place.</p><p>Params are used in different part of notebooks, so best place for param is top of the notebooks.</p>

Reply:
<p>All the parameters will be in one place, except for the <code>path</code>. But right now:</p><ul><li>you need to scroll more to get to parameters section from the results</li><li>the parameters are separated from the instances they parametrize (model, optimizer, etc)</li><li>dataset loading is separated from dataset exploration with model description and parameters</li></ul>